html/changelogs/GeneralCamo - Assorted sprite ports.yml

```
@@ -0,0 +1,43 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes:
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: GeneralCamo
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - imageadd: "Replaced table lamp sprites (Credit to CEV Eris, Kegdo)"
+  - imageadd: "Replaced static air tank sprites (Credit to CEV Eris, SpadesSlick, with minor adjustments by GeneralCamo)"
+  - imageadd: "Replaced Mass Driver sprites (Credit to /tg/station, Wallem)"
```

```suggestion
  - imageadd: "Replaced table lamp sprites."
  - imageadd: "Replaced volume tank sprites."
  - imageadd: "Replaced mass driver sprites."
```
credit is already part of the PR description

---

`CONTRIBUTING.md` guidelines state to put this in the PR description, and the changelog plaintext, or the `author` field.

---

preferably put the co-authors in the author field, then.